.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
```

Comment:
We don't want to publish on every master push, only when we tag a release. Maybe also only from a separate release branch. How is it handled here?